modules/core/src/transactionBuilder.ts

```diff
@@ -898,7 +898,6 @@ exports.signTransaction = function(params) {
   let rootExtKey;
   if (keychain) {
     rootExtKey = bitcoin.HDNode.fromBase58(keychain.xprv);
-    rootExtKeyPath = hdPath(rootExtKey);
```

Comment:
You can remove the `let rootExtKeyPath;` declaration a couple of lines up because it's no longer being used.

Reply:
Looks like this unused var is still there.

Reply:
fixed